bot.user.js

```diff
@@ -922,11 +922,151 @@ var bot = window.bot = (function (window) {
             return false;
         },
 
+        populatePts: function () {
+            let x = window.snake.xx + window.snake.fx;
+            let y = window.snake.yy + window.snake.fy;
```

Comment:
I plotted small circles on (xx,yy) and (xx+fx, yy+fy), and the latter one moved smoother with the snake's head. Yet I din't really understand what they are.